src/trio/_core/_run.py

```diff
@@ -70,19 +76,25 @@
     # for some strange reason Sphinx works with outcome.Outcome, but not Outcome, in
     # start_guest_run. Same with types.FrameType in iter_await_frames
     import outcome
-    from typing_extensions import Self
+    from typing_extensions import Self, TypeVarTuple, Unpack
+
+    PosArgT = TypeVarTuple("PosArgT")
+
+    # Needs to be guarded, since Unpack[] would be evaluated at runtime.
+    class _NurseryStartFunc(Protocol[Unpack[PosArgT], StatusT_co]):
```

Comment:
Curious about why you need an Unpack here

Reply:
Oh sorry, I should have paid more attention to pep 646: https://peps.python.org/pep-0646/#type-variable-tuples-must-always-be-unpacked

Reply:
TypeVarTuple is basically always used in unpacked form. If we were doing 3.11+ only code, it'd be `Protocol[*PosArgT, StatusT_co]`, like we're unpacking the TypeVars inside the TVT.